decidim-core/app/mailers/decidim/reported_mailer.rb

```diff
@@ -21,14 +21,27 @@ def report(user, report)
       end
     end
 
-    def hide(user, report)
+    def hidden_by_admin(user, report, current_user)
```

Comment:
```suggestion
    # This is used when a user with special rights (like an administrator, a space administrator or a moderator) hides a resource
    def hidden_manually(user, report, current_user)
```

Reply:
Suggestion applied in 8555f7f9dd . Done without Github's help as we needed to rename a file.